.github/2.0/cookbooks/Executor.md

```diff
@@ -489,3 +489,45 @@ Line number corresponds to the 1.x code:
     - replacing previous `Blob2PngURI` and `ExcludeQL` driver logic using `Document` built-in
       methods `convert_blob_to_uri` and `pop`
     - there is nothing to return, as the change is done in-place.
+
+
+
+### Scikit-learn
+
+
+
+```python
+import sklearn
+
+from jina import Executor, requests, DocumentArray
+
+class TFIDFTextEncoder(Executor):
+
+    def __init__(
+        self,
+        path_vectorizer: str = os.path.join(cur_dir, 'model/tfidf_vectorizer.pickle'),
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        self.path_vectorizer = path_vectorizer
+
+        import os
+        import pickle
+
+        if os.path.exists(self.path_vectorizer):
+            self.tfidf_vectorizer = pickle.load(open(self.path_vectorizer, 'rb'))
+        else:
```

Comment:
remove this, since it is noisy and will not work.  Let's keep a simple one that load english or something shorter?

Reply:
Let me fit the model there and load the data from sklearn so it is more compact